packages/components/src/avatar/src/Avatar.css

```diff
@@ -65,6 +55,21 @@
     align-items: center;
 }
 
+.o-ui-avatar-group .o-ui-avatar {
+    border: 1px solid var(--o-ui-white);
```

Comment:
White should be replaced with `--hop-samoyed`

Reply:
but shouldn't we be using semantic tokens? not core ones

Reply:
We don't have a semantic border token for that situation as far as I know. It should be the same color as the background.